packages/jazz-tools/src/coValues/coMap.ts

```diff
@@ -151,7 +151,7 @@ export class CoMap extends CoValueBase implements CoValue {
   }
 
   /** @internal */
-  private getEditFromRaw(
+  public getEditFromRaw(
```

Comment:
Typescript was whining about this restricted visibility but I don't have a clear explaination